src/components/TopicsSelect.jsx

```diff
@@ -0,0 +1,21 @@
+function TopicsSelect({ setCurrentPage, setCurrentTopic }) {
+  return (
+    <div className="topics-select">
+      <label htmlFor="topics">Choose a topic:</label>
+      <select
+        name="topics"
+        onChange={(event) => {
+          setCurrentTopic(event.target.value);
+          setCurrentPage(1);
+        }}
+      >
+        <option value="Blank"></option>
```

Comment:
Sorting By Blank isn't great for accessibility, it's not visually obvious either :)